src/gcp_scanner/scanner.py

```diff
@@ -230,6 +230,21 @@ def crawl_loop(initial_sa_tuples: List[Tuple[str, Credentials, List[str]]],
         if dump_file_names is not None:
           dump_file_names.close()
 
+      # Get storage buckets IAM Policy
+      if is_set(scan_config, 'storage_buckets_iam'):
```

Comment:
is there anyway we can avoid the duplication of the code here? We already have most of the stuff above.

Reply:
@mshudrak , I removed 'dump_file_names' code. Can we leave it as it is? Or do you want to merge them?

Reply:
I think we can just move the whole logic under the previous is_set statement where we check for storage_buckets to avoid situation when user wants both fetch bucket names and IAM policies. You need to fetch list of buckets anyway if you want IAM policies, right?